Update the Hardware specs (#298)
NUbots/NUbook: src/book/02-system/04-hardware/01-overview.mdx

The following modifications were made to fit our specific requirements:

- Split components of the robot to fit the size of the printer bed and volume.
- Modified the torso to have open shoulder covers for shoulder servo cable access.

> - Redesigned the legs with a simpler model made out of aluminium, with XH540-W270-R servos rather than MX-106 servos, and onyx connectors for the hip and ankle blocks and onyx supports inside the legs.

Reviewer: ```suggestion
- Redesigned the legs with a simpler model made out of aluminium, with XH540-W270-R servos rather than MX-106 servos, onyx connectors for the hip and ankle blocks and onyx supports inside the legs.
```

- Moved the buttons to the panel above the removable back panel to prevent issues with button cables when removing the back panel.

> - Added an aluminium bar between the hip yaw servos to reduce flex.

Reviewer: ```suggestion
- Added a 3d printed brace between the hip yaw servos to reduce flex. The brace also serves as a mount for power switches and the main power fuse.
```

- Created custom draws for the computer (intel NUC) and subcontrollers (OpenCR, NUSense)

> 

Reviewer: - Designed a bearing housing for the hip yaw that takes a tapered roller bearing from a mountain bike head stock. This interfaces between the servo shaft and the hip pitch/roll block and takes the load of the servo shaft and spreads it over a wider area on the bottom of the torso.

> - Created custom draws for the computer (intel NUC) and subcontrollers (OpenCR, NUSense)

Author: ```suggestion
- Created custom draws for the computer (intel NUC) and subcontrollers (OpenCR, NUSense)
- Designed a bearing housing for the hip yaw that takes a tapered roller bearing from a mountain bike head stock. This interfaces between the servo shaft and the hip pitch/roll block and takes the load of the servo shaft and spreads it over a wider area on the bottom of the torso.
```

## Servos

> The arms and head of the NUgus robot use [Dynamixel MX64AR servos](http://emanual.robotis.com/docs/en/dxl/mx/mx-64/), with two in each shoulder, one in each elbow, and two in the neck. The legs use [Dynamixel XH540-W270-R servo](https://emanual.robotis.com/docs/en/dxl/x/xh540-w270/), with two in each ankle, one in each knee, and three in each hip.

Reviewer: ```suggestion
The arms and head of the NUgus robot use [Dynamixel MX64AR servos](http://emanual.robotis.com/docs/en/dxl/mx/mx-64/), with two in each shoulder, one in each elbow, and two in the neck. The legs use both  [Dynamixel XH540-W270-R servo](https://emanual.robotis.com/docs/en/dxl/x/xh540-w270/) and [Dynamixel MX106 servos](http://emanual.robotis.com/docs/en/dxl/mx/mx-106/), with two X series in each ankle, one in each knee, and two in each hip, with the two hip yaw servos remaining as MX-106.
```

There are custom made mounting brackets milled from aluminium for the [flywheels](https://github.com/NUbots/NUware/blob/main/Igus/Current%20Design/stl_file/Flywheel_Bracket.stl), and servo-to-servo mounting brackets in the [neck](https://github.com/NUbots/NUware/blob/main/Igus/Current%20Design/stl_file/Head_Bracket.stl), hip, and ankle. There are also 3D printed cable clamps on the back of the [knees](https://github.com/NUbots/NUware/blob/main/Igus/Current%20Design/stl_file/Servo_cable_holder.stl), [ankle and hip blocks](https://github.com/NUbots/NUware/blob/main/Igus/Current%20Design/stl_file/Cable_Holder.stl).

> The MX servos have been modified to include a [1.5KE18CA TranZorb](https://www.st.com/content/ccc/resource/technical/document/datasheet/03/7a/4e/3b/96/c3/4e/37/CD00000663.pdf/files/CD00000663.pdf/jcr:content/translations/en.CD00000663.pdf), formerly [1N6385 TranZorb](http://www.vishay.com/docs/88356/icte.pdf), connected in parallel to the servo motor.

Author: ```suggestion
The MX servos have been modified to include a [1.5KE18CA TranZorb](https://www.st.com/content/ccc/resource/technical/document/datasheet/03/7a/4e/3b/96/c3/4e/37/CD00000663.pdf/files/CD00000663.pdf/jcr:content/translations/en.CD00000663.pdf), formerly [1N6385 TranZorb](http://www.vishay.com/docs/88356/icte.pdf), connected in parallel to the servo motor. X series servos do not need a TranZorb.
```

## Networking

The NUgus robot has an Ethernet port as well as WiFi capability. It has two custom-ordered 15cm MHF.4 to RP-SMA wireless antenna cables from the [RFShop](https://rfshop.com.au/), with two [RP-SMA antennas](https://au.element14.com/microchip/rn-sma-s-rp/antenna-2-4ghz-1inch-w-rpsma/dp/2143323) which are around 1" long.

## Power

The NUgus robot has a [Cosel AC/DC Power Supply](https://www.powerbox.com.au/ac-dc-power-supplies/ac-dc-power-supply-cosel-pbb20c). It uses a [4200mAh 4S LiPo Battery](https://www.auselectronicsdirect.com.au/14.8v-4200mah-lipo-4s-battery-pack-with-deans-conn) when not plugged into power. It uses a rebuilt Darwin power board and a custom 16A cylindrical fuse holder. The NUC power board which distributes power to the cooling fan and the NUC is also custom.
The NUgus robot uses custom cables to provide power to the fan, the NUC, and sense and speed control to the fan.

## Foot Studs

> The studs attached to the feet have been purchased from [RS Online](https://au.rs-online.com/web/p/stud-mounts/1264940/?sra=pstk). The Prototyping Workshop in EC can shave the sides of these cylindrical rubber studs into a more conical shape. An M5 nut is needed for attaching the studs to the feet.

Reviewer: ```suggestion
The studs attached to the feet have been purchased from [RS Online](https://au.rs-online.com/web/p/stud-mounts/1264940/?sra=pstk). The Prototyping Workshop in EC can shave the sides of these cylindrical rubber studs into a more conical shape. An M5 nut is needed for attaching the studs to the feet.
An alternative foot stud can be purchased from [AliExpress](https://www.aliexpress.com/item/1005006840189285.html?spm=a2g0o.order_list.order_list_main.17.639c1802I3ZTXL). These are a screw in cleat used on golf shoes and provide excellent grip on synthetic grass. They can also be sourced from most golf shops.
```

## Computing

The NUgus robot uses the [Intel NUC12WSHi7](https://www.intel.com/content/www/us/en/products/sku/121640/intel-nuc-12-pro-kit-nuc12wshi7/specifications.html)

> The USB and fan sense connectors are [Molex 51021-0400](https://au.element14.com/molex/51021-0400/connector-housing-rcpt-4pos/dp/9733116) and the power connectors on the NUC and fan are [Molex 50-37-5043](https://au.element14.com/molex/50-37-5043/connector-rcpt-4pos-1row-2-5mm/dp/9979565). The CM740 to NUC power connector is a [Yeonho SMH250-02](https://www.alldatasheet.com/datasheet-pdf/pdf/148130/YEONHO/SMH250-02.html).

Author: this needs reworking.

Author: ```suggestion
```
